docs/INSTALLATION.md

```diff
@@ -1,15 +1,4 @@
 # Installation
 
-<!-- TODO: write document
-
-  This document should describe what the student needs to install
-  to allow working on the track on their local system using the CLI.
-
-  You can include the installation instructions in this document, but
-  usually it is better to link to a resource with the official installation
-  instructions, to prevent the instructions from becoming outdated.
-
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/installation`.
-
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+To install Roc, please follow the instructions at [roc-lang.org/install](https://roc-lang.org/install).
+You only need to install Roc, and optionally the editor Extensions/Plugins of your choice, you don't need to install Rust or Zig (which are used to build Roc itself).
```

Comment:
Not sure if any student would have expected to install Rust of Zig, that seems more like an implementation detail.
Maybe instead of this:

```suggestion
There are also some [Editor Extensions/Plugins](https://www.roc-lang.org/install#editor-extensions) that you might want to install.
```

Or something like that

Reply:
Hi @ErikSchierboom ,
I'm sorry I missed your last comments and I merged the PR. I'm happy to create another one to address your comments if you want.
@keiravillekode or @BNAndras suggested above that it might be good to clarify exactly what needs to be installed, since Roc's installation instructions give you several options (including installing Zig or Rust, in case you want to build Roc from source or create some Roc platforms).

Reply:
Ah right! Nevermind, forget about my comment then.